lib/scout_apm/config.rb

```diff
@@ -159,7 +159,8 @@ class ConfigDefaults
         'enable_background_jobs' => true,
         'ignore'                 => [],
         'dev_trace'              => false,
-        'profile'                => true # for scoutprof
+        'profile'                => true, # for scoutprof
+        'compress_payload'       => false,
```

Comment:
It will be true for release. for the moment I just don't want to default to breaking things if it does infact break things.